packages/api/controllers/nomination.js

```diff
@@ -77,13 +77,42 @@ const createNomination = async (req, res) => {
     return res.status(500).json({ error: error.message });
   }
 };
+
+const resendEmail = async (req, res) => {
+  const { id } = req.params;
+  const { recipient, emailType } = req.body;
+  try {
+    const nomination = await db.Nomination.findOne({
+      where: { id },
+    });
+    console.log(
+      `email sent to: ${recipient.replace('-', ' ')}`,
+      `email type sent: ${emailType}`
+    );
+    if (recipient === 'family-member' && emailType === 'hipaa') {
+      sendHIPAAEmail(nomination);
+    } else if (recipient === 'family-member' && emailType === 'survey') {
+      sendSurveyEmail(nomination);
+    } else if (recipient === 'healthcare-provider' && emailType === 'hipaa') {
+      sendHIPAAProvider(nomination);
+    } else if (recipient === 'healthcare-provider' && emailType === 'survey') {
+      sendSurveySocialWorker(nomination);
+    } else {
+      return res.status(400).json({ error: '400 error' });
+    }
+  } catch {
+    console.log('404 Not Found', error);
```

Comment:
Not sure a 404 response makes sense here. Probably want to return a 500 if it's an unknown error.

Reply:
fixed!